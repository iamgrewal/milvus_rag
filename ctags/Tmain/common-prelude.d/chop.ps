(abcx) ==
(abcx) _chop pstack clear

(abc ) ==
(abc ) _chop_space pstack clear

(abcx) ==
(abcx) _chop_space pstack clear
