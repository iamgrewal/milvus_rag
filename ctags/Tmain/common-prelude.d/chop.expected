(abcx)
(abc)
(abc )
(abc)
(abcx)
(abcx)
